addons/storyshots/storyshots-core/src/frameworks/angular/loader.js

```diff
@@ -1,13 +1,12 @@
 import 'core-js';
-import 'core-js/es6/reflect';
-import 'core-js/es7/reflect';
+import 'core-js/es/reflect';
 import hasDependency from '../hasDependency';
 import configure from '../configure';
 
 function setupAngularJestPreset() {
   // Needed to prevent "Zone.js has detected that ZoneAwarePromise `(window|global).Promise` has been overwritten."
   require.requireActual('core-js');
-  require.requireActual('core-js/modules/es6.promise');
+  require.requireActual('core-js/modules/es.promise');
   // require.requireActual('core-js/es6/reflect');
   // require.requireActual('core-js/es7/reflect');
```

Comment:
Should we not clean the now outdated commented code about `core-js` like these two?